python/ray/experimental/state/api.py

```diff
@@ -1207,20 +1207,17 @@ def get_log(
         timeout=timeout,
         suffix=suffix,
         attempt_number=attempt_number,
+        ignore_server_stream_error=None,
     )
-    options_dict = {}
-    for field in fields(options):
-        option_val = getattr(options, field.name)
-        if option_val is not None:
-            options_dict[field.name] = option_val
+    options_dict = options.dict()
```

Comment:
```suggestion
    options_dict = options.as_dict()
```
nit comment! (same as dataclass I believe)